pkgs/applications/audio/fmsynth/default.nix

```diff
@@ -0,0 +1,48 @@
+{ stdenv, fetchFromGitHub, gtkmm2, lv2, lvtk, pkgconfig }:
+stdenv.mkDerivation rec {
+  name = "fmsynth-git-${version}";
```

Comment:
git -> unstable

No need to change the attribute in `all-packages.nix` since there is no stable version.